packages/webapp/src/delegates/api/fixtures.ts

```diff
@@ -4,7 +4,7 @@ export const fixtureMembersStats: MemberStats = {
     active_members: 4,
     pending_members: 0,
     completed_waiting_inductions: 0,
-    ranks: [4, 1],
+    ranks: [1, 4, 1],
 };
 
 export const fixtureMembers: MemberData[] = [
```

Comment:
```
export const fixtureMembersStats: MemberStats = {
    active_members: 4,
    pending_members: 0,
    completed_waiting_inductions: 0,
    ranks: [1, 4, 1],
};
```

There are 4 active_members. But there are 6 people in the ranks. And then we have 5 active members in the members fixture. Am I reading that right?

Should we consider calculating the values in `fixtureMembersStats` from the values in `fixtureMembers` so that they're always guaranteed to be in sync (assuming we use `fixtureMembers` as the source of truth and derive `fixtureMemberStats` from it?) Or maybe you found that it's better to keep them decoupled and separate so we can more nimbly tweak values? Just want to understand better your thinking on that.

Reply:
I've updated the fixtures to be more consistent with notes to describe the dependencies. They were written as I developed each section, so they weren't ever meant to be completely consistent, but I got them all pretty close (with the caveats in comments)